gcn/notices/neutrino/Alert.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/neutrino/Alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "Neutrino Alert",
+  "description": "Neutrino alert common base schema for high-energy neutrino telescope alerts.  To be inherited from for alerts on experiment specific alert topics",
+  "allOf": [
+    {
+      "$ref": "../core/Event.schema.json",
+      "description": "Unique per-alert identification information from source telescope"
+    },
+    {
+      "$ref": "../core/Alert.schema.json",
+      "description": "Alert information"
+    },
+    {
+      "$ref": "../core/Reporter.schema.json",
+      "description": "Reporting telescope ID information"
+    },
+    {
+      "$ref": "../core/Statistics.schema.json",
+      "description": "Alert significance measures"
+    },
+    {
+      "$ref": "../core/Localization.schema.json",
+      "description": "Alert sky localization information"
+    },
+    { "$ref": "../core/DateTime.schema.json" }
+  ],
+  "properties": {
+    "number_of_events": {
+      "description": "Number of detected events that contributed to alert",
+      "type": "number"
+    },
+    "alert_topology": {
+      "description": "Event topology of the alert",
+      "enum": ["Track", "Shower", "Multiplet"]
+    }
```

Comment:
Is this really called topology? It sounds like morphology.

Reply:
It is the standard terminology in the field. e.g.
https://gcn.gsfc.nasa.gov/doc/High_Energy_Neutrino_Cascade_Alerts.pdf
https://icecube.wisc.edu/news/research/2020/02/new-insights-into-astrophysical-neutrino-flux/
https://arxiv.org/abs/1903.04175

Reply:
Weird. Thanks for checking.